src/components/Nav/NewCollectiveModal.vue

```diff
@@ -87,6 +87,7 @@
 					<MemberPicker :show-selection="true"
 						:selected-members="selectedMembers"
 						:on-click-searched="onClickSearched"
+						:empty-content-hint="t('collectives', 'Your user is added automatically.')"
```

Comment:
Not sure about the wording. Maybe "As creator, you're already added." is better? :thinking: